recipes/iverilog/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "10.1" %}
+{% set git_tag = "v10_1" %}
+
+package:
+  name: iverilog
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/steveicarus/iverilog.git
+  git_tag: {{ git_tag }}
+  patches:
+    - remove_static_build_path.patch
```

Comment:
Please reference the PR in a comment above this patch so that it will be easy to track when we can drop it.